### DISCUSSION TITLE
How to customise/edit the Admin UI styling

### DISCUSSION THREAD

**Discussion by Vopa37** [keystonejs/keystone#7343] [Questions] [answered]
Hi everybody. Is it able to edit the administration page in keystone-6? I have problem with page components dropdown menu (its hidden behind the submit panel down the component) 
<img width="837" alt="image" src="https://user-images.githubusercontent.com/62169884/155706312-bf5ddf2b-5064-4fa5-ab84-bf01c67d61c7.png">

**Comment by gwyneplaine** [comment, marked as answer]
Thanks for raising this @Vopa37  this is absolutely a bug, I've created a requisite issue here #7395  to track 👍
